defaults/main.yml

```
@@ -1 +1,13 @@
 clamav_cron_frequency: weekly
+clamav_scan_exclude_directories:
+  - ^/dev/
+  - ^/sys/
+  - ^/proc/
+  - ^/var/spool/clamav
```

Are these `"^"` required here?  If not, we should remove them:
```suggestion
  - /dev/
  - /proc/
  - /sys/
  - /var/spool/clamav
```

---

Actually these are required, as reported in #50 and documented in the [clamscan man page](https://linux.die.net/man/1/clamscan).

---

Thanks- I had not seen #50. 👍